Entities/Characters/Knight/KnightLogic.as

```diff
@@ -1154,6 +1150,30 @@ void onCommand(CBlob@ this, u8 cmd, CBitStream @params)
 	{
 		SetFirstAvailableBomb(this);
 	}
+	else if (cmd == this.getCommandID("activate/throw bomb"))
```

Comment:
I'm a bit confused: `"get bomb"` cannot be sent straight from the client anymore, correct? If `"activate/throw bomb"` is run on the serverside, why do we need to fire a `"get bomb"` command as opposed to just calling the relevant code directly?

This is not just for style FWIW, because I think the fired command will only be effective after a tick, but I'm not sure.

Reply:
fixed